conjur/logic/credential_provider/keystore_credentials_provider.py

```diff
@@ -80,6 +82,35 @@ def remove_credentials(self, conjurrc):
         """
         Method for removing user credentials in the system's keyring
         """
-        logging.debug(f"Attempting to remove credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
-        KeystoreAdapter.delete_password(conjurrc.conjur_url)
-        logging.debug(f"Successfully removed credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        logging.debug("Attempting to remove credentials from "
+                      f"the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        for attr in KEYSTORE_ATTRIBUTES:
+            try:
+                KeystoreAdapter.delete_password(conjurrc.conjur_url, attr)
+            # Catches when credentials do not exist in the keyring. If the key does not exist,
+            # the user has already logged out. we still try to remove other leftovers
+            except KeyringAdapterDeletionError:
+                logging.debug(
+                    f"Unable to delete item '{attr}' from the '{KeystoreAdapter.get_keyring_name()}' "
```

Comment:
logging.debug(f"Unable to delete item '{attr}' from the '{KeystoreAdapter.get_keyring_name()}' "
                       f"credential store. Item may not exist.\n {traceback.format_exc()}"))